src/device/router_device_worker.erl

```diff
@@ -374,18 +374,28 @@ handle_cast(
                         []
                 end,
 
-            DeviceUpdates = [
-                {name, router_device:name(APIDevice)},
-                {dev_eui, router_device:dev_eui(APIDevice)},
-                {app_eui, router_device:app_eui(APIDevice)},
-                {devaddrs, DevAddrs},
-                {metadata,
-                    maps:merge(
-                        lorawan_rxdelay:maybe_update(APIDevice, Device0),
-                        router_device:metadata(APIDevice)
-                    )},
-                {is_active, IsActive}
-            ],
+            ECCUpdate =
+                case router_device:ecc_compact(APIDevice) of
+                    undefined ->
+                        [];
+                    ECC ->
```

Comment:
From the device record.
`ecc_compact :: map() | undefined,`

I think that's going to break `base64:decode/1`